### ISSUE TITLE
Spree 2.1 - Error when trying to access Products Groups in admin

### ISSUE THREAD

**Issue by fonemstr** [spree-contrib/spree_product_groups#28]
On a fresh install of Spree 2.1 getting the following error when trying to access Product Groups in admin:

undefined method `relation' for #ActiveRecord::Relation::ActiveRecord_Relation_Spree_Product:0x007fdabb387f00

**Comment by radar** [comment]
@fonemstr Which branch of this extension are you using?

**Comment by fonemstr** [comment]
I'm using 2-1-stable. I messed up...the issue stated is actually what I'm getting from my recently updated store and not from a fresh install. On a fresh install (2-1-stable)  you can access product groups in admin and  create a product group but when you try to add a scope you get a ActiveModel::ForbiddenAttributesError (ActiveModel::ForbiddenAttributesError) from the product_scopes_controller.rb in 'create'

**Comment by raulpopadineti** [comment]
Same issue here. I was able to create a product group, but when trying to add a scope I get: 

``` ruby
undefined method `relation' for #<ActiveRecord::Relation::ActiveRecord_Relation_Spree_Product:0x007fb1442fa588>
```

**Comment by radar** [comment]
@raulpopadineti please provide the information we ask for in the [Contributing Guidelines](https://github.com/spree/spree/tree/master/CONTRIBUTING.md).

Thanks!

**Comment by raulpopadineti** [comment]
I wasn't able to reproduce the error, but now when I'm trying to add a product group, after create, if I want to add the default scope "In taxons and all their descendants", I receive the error:

``` ruby
ActionController::InvalidAuthenticityToken
```

I'm using the 2-1-stable branch from spree and rails version 4.0.

Here is my complete Gemfile:

``` ruby
source 'https://rubygems.org'

gem 'rails', '4.0.0'

gem 'mysql2'

gem 'sass-rails', '~> 4.0.0'

gem 'uglifier', '>= 1.3.0'

gem 'coffee-rails', '~> 4.0.0'

gem 'compass-rails', github: 'Compass/compass-rails', branch: 'rails4-hack'

# gem 'therubyracer', platforms: :ruby

gem 'jquery-rails'

gem 'jquery-ui-rails'

gem 'turbolinks'

gem 'jbuilder', '~> 1.2'

gem 'devise-i18n'

gem 'mobile-fu'

gem 'annotate', ">=2.5.0"

gem 'whenever', :require => false

group :development, :test do
  gem 'thin'
  gem 'quiet_assets'
  gem 'better_errors'
  gem 'mailcatcher'
  gem 'binding_of_caller'
end

group :doc do
  # bundle exec rake doc:rails generates the API under doc/api.
  gem 'sdoc', require: false
end

# Use ActiveModel has_secure_password
# gem 'bcrypt-ruby', '~> 3.0.0'

# Use unicorn as the app server
gem 'unicorn'

# Use Capistrano for deployment
group :development do
  gem 'capistrano', '~> 3.0.0'
  gem 'capistrano-bundler'
end

# Use debugger
# gem 'debugger', group: [:development, :test]

gem 'spree', github: 'spree/spree', branch: '2-1-stable'

gem 'spree_i18n', github: 'spree/spree_i18n', branch: '2-1-stable'

gem 'spree_auth_devise', github: 'spree/spree_auth_devise'

gem 'spree_zero_stock_products', '~> 2.1.0'

gem 'spree_static_content', github: 'spree/spree_static_content'

gem 'spree_reviews', github: 'spree/spree_reviews', branch: '2-1-stable'

gem 'spree_social', github: 'spree/spree_social', branch: '2-1-stable'

gem 'spree_product_groups', github: 'spree/spree_product_groups', branch: '2-1-stable', ref: '937098de2aa0065a4eea97e695ca6b8e8ae36314'

gem 'spree_slider'

gem 'spree_sitemap', github: 'jdutil/spree_sitemap'

gem 'spree_mailchimp_gibbon', github: 'raulpopadineti/spree_mailchimp_gibbon', branch: '2-1-fix', ref: '85af7fe1928b653f4cd7f95edae430b5869b314e'
```

and here is the full-stack trace:

``` ruby
ActionController::InvalidAuthenticityToken - ActionController::InvalidAuthenticityToken:
  actionpack (4.0.0) lib/action_controller/metal/request_forgery_protection.rb:163:in `handle_unverified_request'
  actionpack (4.0.0) lib/action_controller/metal/request_forgery_protection.rb:170:in `handle_unverified_request'
  devise (3.0.2) lib/devise/controllers/helpers.rb:278:in `handle_unverified_request'
  actionpack (4.0.0) lib/action_controller/metal/request_forgery_protection.rb:177:in `verify_authenticity_token'
  activesupport (4.0.0) lib/active_support/callbacks.rb:377:in `_run__540121438163223319__process_action__callbacks'
  activesupport (4.0.0) lib/active_support/callbacks.rb:80:in `run_callbacks'
  actionpack (4.0.0) lib/abstract_controller/callbacks.rb:17:in `process_action'
  actionpack (4.0.0) lib/action_controller/metal/rescue.rb:29:in `process_action'
  actionpack (4.0.0) lib/action_controller/metal/instrumentation.rb:31:in `block in process_action'
  activesupport (4.0.0) lib/active_support/notifications.rb:159:in `block in instrument'
  activesupport (4.0.0) lib/active_support/notifications/instrumenter.rb:20:in `instrument'
  activesupport (4.0.0) lib/active_support/notifications.rb:159:in `instrument'
  actionpack (4.0.0) lib/action_controller/metal/instrumentation.rb:30:in `process_action'
  actionpack (4.0.0) lib/action_controller/metal/params_wrapper.rb:245:in `process_action'
  activerecord (4.0.0) lib/active_record/railties/controller_runtime.rb:18:in `process_action'
  actionpack (4.0.0) lib/abstract_controller/base.rb:136:in `process'
  actionpack (4.0.0) lib/abstract_controller/rendering.rb:44:in `process'
  actionpack (4.0.0) lib/action_controller/metal.rb:195:in `dispatch'
  actionpack (4.0.0) lib/action_controller/metal/rack_delegation.rb:13:in `dispatch'
  actionpack (4.0.0) lib/action_controller/metal.rb:231:in `block in action'
  actionpack (4.0.0) lib/action_dispatch/routing/route_set.rb:80:in `dispatch'
  actionpack (4.0.0) lib/action_dispatch/routing/route_set.rb:48:in `call'
  actionpack (4.0.0) lib/action_dispatch/journey/router.rb:71:in `block in call'
  actionpack (4.0.0) lib/action_dispatch/journey/router.rb:59:in `call'
  actionpack (4.0.0) lib/action_dispatch/routing/route_set.rb:655:in `call'
  railties (4.0.0) lib/rails/engine.rb:511:in `call'
  railties (4.0.0) lib/rails/railtie/configurable.rb:30:in `method_missing'
  actionpack (4.0.0) lib/action_dispatch/journey/router.rb:71:in `block in call'
  actionpack (4.0.0) lib/action_dispatch/journey/router.rb:59:in `call'
  actionpack (4.0.0) lib/action_dispatch/routing/route_set.rb:655:in `call'
  omniauth (1.1.4) lib/omniauth/strategy.rb:184:in `call!'
  omniauth (1.1.4) lib/omniauth/strategy.rb:164:in `call'
  omniauth (1.1.4) lib/omniauth/strategy.rb:184:in `call!'
  omniauth (1.1.4) lib/omniauth/strategy.rb:164:in `call'
  omniauth (1.1.4) lib/omniauth/strategy.rb:184:in `call!'
  omniauth (1.1.4) lib/omniauth/strategy.rb:164:in `call'
  omniauth (1.1.4) lib/omniauth/strategy.rb:184:in `call!'
  omniauth (1.1.4) lib/omniauth/strategy.rb:164:in `call'
  rack-mobile-detect (0.4.0) lib/rack/mobile-detect.rb:164:in `call'
  warden (1.2.3) lib/warden/manager.rb:35:in `block in call'
  warden (1.2.3) lib/warden/manager.rb:34:in `call'
  rack (1.5.2) lib/rack/etag.rb:23:in `call'
  rack (1.5.2) lib/rack/conditionalget.rb:35:in `call'
  rack (1.5.2) lib/rack/head.rb:11:in `call'
  actionpack (4.0.0) lib/action_dispatch/middleware/params_parser.rb:27:in `call'
  actionpack (4.0.0) lib/action_dispatch/middleware/flash.rb:241:in `call'
  rack (1.5.2) lib/rack/session/abstract/id.rb:225:in `context'
  rack (1.5.2) lib/rack/session/abstract/id.rb:220:in `call'
  actionpack (4.0.0) lib/action_dispatch/middleware/cookies.rb:486:in `call'
  activerecord (4.0.0) lib/active_record/query_cache.rb:36:in `call'
  activerecord (4.0.0) lib/active_record/connection_adapters/abstract/connection_pool.rb:626:in `call'
  activerecord (4.0.0) lib/active_record/migration.rb:369:in `call'
  actionpack (4.0.0) lib/action_dispatch/middleware/callbacks.rb:29:in `block in call'
  activesupport (4.0.0) lib/active_support/callbacks.rb:373:in `_run__1778482659195044036__call__callbacks'
  activesupport (4.0.0) lib/active_support/callbacks.rb:80:in `run_callbacks'
  actionpack (4.0.0) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (4.0.0) lib/action_dispatch/middleware/reloader.rb:64:in `call'
  actionpack (4.0.0) lib/action_dispatch/middleware/remote_ip.rb:76:in `call'
  better_errors (1.0.1) lib/better_errors/middleware.rb:84:in `protected_app_call'
  better_errors (1.0.1) lib/better_errors/middleware.rb:79:in `better_errors_call'
  better_errors (1.0.1) lib/better_errors/middleware.rb:56:in `call'
  actionpack (4.0.0) lib/action_dispatch/middleware/debug_exceptions.rb:17:in `call'
  actionpack (4.0.0) lib/action_dispatch/middleware/show_exceptions.rb:30:in `call'
  railties (4.0.0) lib/rails/rack/logger.rb:38:in `call_app'
  railties (4.0.0) lib/rails/rack/logger.rb:21:in `block in call'
  activesupport (4.0.0) lib/active_support/tagged_logging.rb:67:in `block in tagged'
  activesupport (4.0.0) lib/active_support/tagged_logging.rb:25:in `tagged'
  activesupport (4.0.0) lib/active_support/tagged_logging.rb:67:in `tagged'
  railties (4.0.0) lib/rails/rack/logger.rb:21:in `call'
  quiet_assets (1.0.2) lib/quiet_assets.rb:18:in `call_with_quiet_assets'
  actionpack (4.0.0) lib/action_dispatch/middleware/request_id.rb:21:in `call'
  rack (1.5.2) lib/rack/methodoverride.rb:21:in `call'
  rack (1.5.2) lib/rack/runtime.rb:17:in `call'
  activesupport (4.0.0) lib/active_support/cache/strategy/local_cache.rb:83:in `call'
  rack (1.5.2) lib/rack/lock.rb:17:in `call'
  actionpack (4.0.0) lib/action_dispatch/middleware/static.rb:64:in `call'
  railties (4.0.0) lib/rails/engine.rb:511:in `call'
  railties (4.0.0) lib/rails/application.rb:97:in `call'
  rack (1.5.2) lib/rack/content_length.rb:14:in `call'
  thin (1.6.0) lib/thin/connection.rb:82:in `block in pre_process'
  thin (1.6.0) lib/thin/connection.rb:80:in `pre_process'
  thin (1.6.0) lib/thin/connection.rb:55:in `process'
  thin (1.6.0) lib/thin/connection.rb:41:in `receive_data'
  eventmachine (1.0.3) lib/eventmachine.rb:187:in `run'
  thin (1.6.0) lib/thin/backends/base.rb:73:in `start'
  thin (1.6.0) lib/thin/server.rb:162:in `start'
  rack (1.5.2) lib/rack/handler/thin.rb:16:in `run'
  rack (1.5.2) lib/rack/server.rb:264:in `start'
  railties (4.0.0) lib/rails/commands/server.rb:84:in `start'
  railties (4.0.0) lib/rails/commands.rb:78:in `block in <top (required)>'
  railties (4.0.0) lib/rails/commands.rb:73:in `<top (required)>'
  bin/rails:4:in `<top (required)>'
  -e:1:in `<main>'
```

Also when I'm trying to update the sort order, I receive:

``` ruby
wrong number of arguments (1 for 0)
```

Here is the full-stack trace for the update error:

``` ruby
ArgumentError - wrong number of arguments (1 for 0):
  activerecord-deprecated_finders (1.0.3) lib/active_record/deprecated_finders/base.rb:28:in `call'
  activerecord (4.0.0) lib/active_record/scoping/named.rb:163:in `block (2 levels) in scope'
  activerecord (4.0.0) lib/active_record/relation.rb:270:in `scoping'
  activerecord (4.0.0) lib/active_record/scoping/named.rb:163:in `block in scope'
  activerecord (4.0.0) lib/active_record/relation/delegation.rb:36:in `block in descend_by_name'
  activerecord (4.0.0) lib/active_record/relation.rb:270:in `scoping'
  activerecord (4.0.0) lib/active_record/relation/delegation.rb:36:in `descend_by_name'
   () Users/ma3x/.rvm/gems/ruby-2.0.0-p247@bigweld-2.0/bundler/gems/spree_product_groups-937098de2aa0/app/models/spree/product_scope.rb:43:in `apply_on'
   () Users/ma3x/.rvm/gems/ruby-2.0.0-p247@bigweld-2.0/bundler/gems/spree_product_groups-937098de2aa0/app/models/spree/product_group.rb:140:in `block in products'
   () Users/ma3x/.rvm/gems/ruby-2.0.0-p247@bigweld-2.0/bundler/gems/spree_product_groups-937098de2aa0/app/models/spree/product_group.rb:139:in `products'
   () Users/ma3x/.rvm/gems/ruby-2.0.0-p247@bigweld-2.0/bundler/gems/spree_product_groups-937098de2aa0/app/models/spree/product_group.rb:173:in `generate_preview'
   () Users/ma3x/.rvm/gems/ruby-2.0.0-p247@bigweld-2.0/bundler/gems/spree_product_groups-937098de2aa0/app/views/spree/admin/product_groups/_preview.html.erb:1:in `_a71de6fd9eaac4232dd6b9a115563e83'
  actionpack (4.0.0) lib/action_view/template.rb:143:in `block in render'
  activesupport (4.0.0) lib/active_support/notifications.rb:161:in `instrument'
  actionpack (4.0.0) lib/action_view/template.rb:141:in `render'
  deface (1.0.0.rc4) lib/deface/action_view_extensions.rb:41:in `render'
  actionpack (4.0.0) lib/action_view/renderer/partial_renderer.rb:306:in `render_partial'
  actionpack (4.0.0) lib/action_view/renderer/partial_renderer.rb:279:in `block in render'
  actionpack (4.0.0) lib/action_view/renderer/abstract_renderer.rb:38:in `block in instrument'
  activesupport (4.0.0) lib/active_support/notifications.rb:159:in `block in instrument'
  activesupport (4.0.0) lib/active_support/notifications/instrumenter.rb:20:in `instrument'
  activesupport (4.0.0) lib/active_support/notifications.rb:159:in `instrument'
  actionpack (4.0.0) lib/action_view/renderer/abstract_renderer.rb:38:in `instrument'
  actionpack (4.0.0) lib/action_view/renderer/partial_renderer.rb:278:in `render'
  actionpack (4.0.0) lib/action_view/renderer/renderer.rb:47:in `render_partial'
  actionpack (4.0.0) lib/action_view/renderer/renderer.rb:21:in `render'
  actionpack (4.0.0) lib/action_view/helpers/rendering_helper.rb:24:in `render'
   () Users/ma3x/.rvm/gems/ruby-2.0.0-p247@bigweld-2.0/bundler/gems/spree_product_groups-937098de2aa0/app/views/spree/admin/product_groups/edit.html.erb:58:in `_910c6417adebb2ae7db8097dae51cef9'
  actionpack (4.0.0) lib/action_view/template.rb:143:in `block in render'
  activesupport (4.0.0) lib/active_support/notifications.rb:161:in `instrument'
  actionpack (4.0.0) lib/action_view/template.rb:141:in `render'
  deface (1.0.0.rc4) lib/deface/action_view_extensions.rb:41:in `render'
  actionpack (4.0.0) lib/action_view/renderer/template_renderer.rb:49:in `block (2 levels) in render_template'
  actionpack (4.0.0) lib/action_view/renderer/abstract_renderer.rb:38:in `block in instrument'
  activesupport (4.0.0) lib/active_support/notifications.rb:159:in `block in instrument'
  activesupport (4.0.0) lib/active_support/notifications/instrumenter.rb:20:in `instrument'
  activesupport (4.0.0) lib/active_support/notifications.rb:159:in `instrument'
  actionpack (4.0.0) lib/action_view/renderer/abstract_renderer.rb:38:in `instrument'
  actionpack (4.0.0) lib/action_view/renderer/template_renderer.rb:48:in `block in render_template'
  actionpack (4.0.0) lib/action_view/renderer/template_renderer.rb:56:in `render_with_layout'
  actionpack (4.0.0) lib/action_view/renderer/template_renderer.rb:47:in `render_template'
  actionpack (4.0.0) lib/action_view/renderer/template_renderer.rb:17:in `render'
  actionpack (4.0.0) lib/action_view/renderer/renderer.rb:42:in `render_template'
  actionpack (4.0.0) lib/action_view/renderer/renderer.rb:23:in `render'
  actionpack (4.0.0) lib/abstract_controller/rendering.rb:127:in `_render_template'
  actionpack (4.0.0) lib/action_controller/metal/streaming.rb:219:in `_render_template'
  actionpack (4.0.0) lib/abstract_controller/rendering.rb:120:in `render_to_body'
  actionpack (4.0.0) lib/action_controller/metal/rendering.rb:33:in `render_to_body'
  actionpack (4.0.0) lib/action_controller/metal/renderers.rb:26:in `render_to_body'
  actionpack (4.0.0) lib/abstract_controller/rendering.rb:97:in `render'
  actionpack (4.0.0) lib/action_controller/metal/rendering.rb:16:in `render'
  actionpack (4.0.0) lib/action_controller/metal/instrumentation.rb:41:in `block (2 levels) in render'
  activesupport (4.0.0) lib/active_support/core_ext/benchmark.rb:12:in `block in ms'
  /Users/ma3x/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/benchmark.rb:296:in `realtime'
  activesupport (4.0.0) lib/active_support/core_ext/benchmark.rb:12:in `ms'
  actionpack (4.0.0) lib/action_controller/metal/instrumentation.rb:41:in `block in render'
  actionpack (4.0.0) lib/action_controller/metal/instrumentation.rb:84:in `cleanup_view_runtime'
  activerecord (4.0.0) lib/active_record/railties/controller_runtime.rb:25:in `cleanup_view_runtime'
  actionpack (4.0.0) lib/action_controller/metal/instrumentation.rb:40:in `render'
  actionpack (4.0.0) lib/action_controller/metal/responder.rb:136:in `render'
  actionpack (4.0.0) lib/action_controller/metal/responder.rb:192:in `navigation_behavior'
  actionpack (4.0.0) lib/action_controller/metal/responder.rb:163:in `rescue in to_html'
  actionpack (4.0.0) lib/action_controller/metal/responder.rb:161:in `to_html'
   () Users/ma3x/.rvm/gems/ruby-2.0.0-p247@bigweld-2.0/bundler/gems/spree-e9bcb55295c5/core/lib/spree/responder.rb:19:in `to_html'
  actionpack (4.0.0) lib/action_controller/metal/responder.rb:154:in `respond'
  actionpack (4.0.0) lib/action_controller/metal/responder.rb:147:in `call'
   () Users/ma3x/.rvm/gems/ruby-2.0.0-p247@bigweld-2.0/bundler/gems/spree-e9bcb55295c5/core/lib/spree/core/controller_helpers/respond_with.rb:20:in `respond_with'
   () Users/ma3x/.rvm/gems/ruby-2.0.0-p247@bigweld-2.0/bundler/gems/spree-e9bcb55295c5/backend/app/controllers/spree/admin/resource_controller.rb:37:in `update'
  actionpack (4.0.0) lib/action_controller/metal/implicit_render.rb:4:in `send_action'
  actionpack (4.0.0) lib/abstract_controller/base.rb:189:in `process_action'
  actionpack (4.0.0) lib/action_controller/metal/rendering.rb:10:in `process_action'
  actionpack (4.0.0) lib/abstract_controller/callbacks.rb:18:in `block in process_action'
  activesupport (4.0.0) lib/active_support/callbacks.rb:503:in `_run__1757249660185367776__process_action__callbacks'
  activesupport (4.0.0) lib/active_support/callbacks.rb:80:in `run_callbacks'
  actionpack (4.0.0) lib/abstract_controller/callbacks.rb:17:in `process_action'
  actionpack (4.0.0) lib/action_controller/metal/rescue.rb:29:in `process_action'
  actionpack (4.0.0) lib/action_controller/metal/instrumentation.rb:31:in `block in process_action'
  activesupport (4.0.0) lib/active_support/notifications.rb:159:in `block in instrument'
  activesupport (4.0.0) lib/active_support/notifications/instrumenter.rb:20:in `instrument'
  activesupport (4.0.0) lib/active_support/notifications.rb:159:in `instrument'
  actionpack (4.0.0) lib/action_controller/metal/instrumentation.rb:30:in `process_action'
  actionpack (4.0.0) lib/action_controller/metal/params_wrapper.rb:245:in `process_action'
  activerecord (4.0.0) lib/active_record/railties/controller_runtime.rb:18:in `process_action'
  actionpack (4.0.0) lib/abstract_controller/base.rb:136:in `process'
  actionpack (4.0.0) lib/abstract_controller/rendering.rb:44:in `process'
  actionpack (4.0.0) lib/action_controller/metal.rb:195:in `dispatch'
  actionpack (4.0.0) lib/action_controller/metal/rack_delegation.rb:13:in `dispatch'
  actionpack (4.0.0) lib/action_controller/metal.rb:231:in `block in action'
  actionpack (4.0.0) lib/action_dispatch/routing/route_set.rb:80:in `dispatch'
  actionpack (4.0.0) lib/action_dispatch/routing/route_set.rb:48:in `call'
  actionpack (4.0.0) lib/action_dispatch/journey/router.rb:71:in `block in call'
  actionpack (4.0.0) lib/action_dispatch/journey/router.rb:59:in `call'
  actionpack (4.0.0) lib/action_dispatch/routing/route_set.rb:655:in `call'
  railties (4.0.0) lib/rails/engine.rb:511:in `call'
  railties (4.0.0) lib/rails/railtie/configurable.rb:30:in `method_missing'
  actionpack (4.0.0) lib/action_dispatch/journey/router.rb:71:in `block in call'
  actionpack (4.0.0) lib/action_dispatch/journey/router.rb:59:in `call'
  actionpack (4.0.0) lib/action_dispatch/routing/route_set.rb:655:in `call'
  omniauth (1.1.4) lib/omniauth/strategy.rb:184:in `call!'
  omniauth (1.1.4) lib/omniauth/strategy.rb:164:in `call'
  omniauth (1.1.4) lib/omniauth/strategy.rb:184:in `call!'
  omniauth (1.1.4) lib/omniauth/strategy.rb:164:in `call'
  omniauth (1.1.4) lib/omniauth/strategy.rb:184:in `call!'
  omniauth (1.1.4) lib/omniauth/strategy.rb:164:in `call'
  omniauth (1.1.4) lib/omniauth/strategy.rb:184:in `call!'
  omniauth (1.1.4) lib/omniauth/strategy.rb:164:in `call'
  rack-mobile-detect (0.4.0) lib/rack/mobile-detect.rb:164:in `call'
  warden (1.2.3) lib/warden/manager.rb:35:in `block in call'
  warden (1.2.3) lib/warden/manager.rb:34:in `call'
  rack (1.5.2) lib/rack/etag.rb:23:in `call'
  rack (1.5.2) lib/rack/conditionalget.rb:35:in `call'
  rack (1.5.2) lib/rack/head.rb:11:in `call'
  actionpack (4.0.0) lib/action_dispatch/middleware/params_parser.rb:27:in `call'
  actionpack (4.0.0) lib/action_dispatch/middleware/flash.rb:241:in `call'
  rack (1.5.2) lib/rack/session/abstract/id.rb:225:in `context'
  rack (1.5.2) lib/rack/session/abstract/id.rb:220:in `call'
  actionpack (4.0.0) lib/action_dispatch/middleware/cookies.rb:486:in `call'
  activerecord (4.0.0) lib/active_record/query_cache.rb:36:in `call'
  activerecord (4.0.0) lib/active_record/connection_adapters/abstract/connection_pool.rb:626:in `call'
  activerecord (4.0.0) lib/active_record/migration.rb:369:in `call'
  actionpack (4.0.0) lib/action_dispatch/middleware/callbacks.rb:29:in `block in call'
  activesupport (4.0.0) lib/active_support/callbacks.rb:373:in `_run__1778482659195044036__call__callbacks'
  activesupport (4.0.0) lib/active_support/callbacks.rb:80:in `run_callbacks'
  actionpack (4.0.0) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (4.0.0) lib/action_dispatch/middleware/reloader.rb:64:in `call'
  actionpack (4.0.0) lib/action_dispatch/middleware/remote_ip.rb:76:in `call'
  better_errors (1.0.1) lib/better_errors/middleware.rb:84:in `protected_app_call'
  better_errors (1.0.1) lib/better_errors/middleware.rb:79:in `better_errors_call'
  better_errors (1.0.1) lib/better_errors/middleware.rb:56:in `call'
  actionpack (4.0.0) lib/action_dispatch/middleware/debug_exceptions.rb:17:in `call'
  actionpack (4.0.0) lib/action_dispatch/middleware/show_exceptions.rb:30:in `call'
  railties (4.0.0) lib/rails/rack/logger.rb:38:in `call_app'
  railties (4.0.0) lib/rails/rack/logger.rb:21:in `block in call'
  activesupport (4.0.0) lib/active_support/tagged_logging.rb:67:in `block in tagged'
  activesupport (4.0.0) lib/active_support/tagged_logging.rb:25:in `tagged'
  activesupport (4.0.0) lib/active_support/tagged_logging.rb:67:in `tagged'
  railties (4.0.0) lib/rails/rack/logger.rb:21:in `call'
  quiet_assets (1.0.2) lib/quiet_assets.rb:18:in `call_with_quiet_assets'
  actionpack (4.0.0) lib/action_dispatch/middleware/request_id.rb:21:in `call'
  rack (1.5.2) lib/rack/methodoverride.rb:21:in `call'
  rack (1.5.2) lib/rack/runtime.rb:17:in `call'
  activesupport (4.0.0) lib/active_support/cache/strategy/local_cache.rb:83:in `call'
  rack (1.5.2) lib/rack/lock.rb:17:in `call'
  actionpack (4.0.0) lib/action_dispatch/middleware/static.rb:64:in `call'
  railties (4.0.0) lib/rails/engine.rb:511:in `call'
  railties (4.0.0) lib/rails/application.rb:97:in `call'
  rack (1.5.2) lib/rack/content_length.rb:14:in `call'
  thin (1.6.0) lib/thin/connection.rb:82:in `block in pre_process'
  thin (1.6.0) lib/thin/connection.rb:80:in `pre_process'
  thin (1.6.0) lib/thin/connection.rb:55:in `process'
  thin (1.6.0) lib/thin/connection.rb:41:in `receive_data'
  eventmachine (1.0.3) lib/eventmachine.rb:187:in `run'
  thin (1.6.0) lib/thin/backends/base.rb:73:in `start'
  thin (1.6.0) lib/thin/server.rb:162:in `start'
  rack (1.5.2) lib/rack/handler/thin.rb:16:in `run'
  rack (1.5.2) lib/rack/server.rb:264:in `start'
  railties (4.0.0) lib/rails/commands/server.rb:84:in `start'
  railties (4.0.0) lib/rails/commands.rb:78:in `block in <top (required)>'
  railties (4.0.0) lib/rails/commands.rb:73:in `<top (required)>'
  bin/rails:4:in `<top (required)>'
  -e:1:in `<main>'
```
